docs/sql-ref-syntax-qry-select-setops.md

```diff
@@ -19,4 +19,125 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+Set operators are used to combine the results of two queries into a single result. Spark SQL supports three types of set operators:
+- `EXCEPT` and `EXCEPT ALL`
+- `INTERSECT` and `INTERSECT ALL`
+- `UNION` and `UNION ALL`
+
+Note that the queries' result sets must have the same number of columns and compatible data types for the respective columns.
```

Comment:
`Note that the queries' result sets must have...` -> `Note that input relations must have...`?